### PR TITLE
ColorFIx

### PR DESCRIPTION
**Pull request by kwarnerredhat**

### CODE REVIEW

File: src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -95,11 +95,12 @@ export default () => {
               onToggle={() => setSelectToggle(!selectToggle)}
               isOpen={selectToggle}
               onSelect={(_e, value) => {
-                if (typeof value === 'string')
-                  // TODO: Fix this when creating the new Color type
-                  setNewUser({ ...newUser, color: value });
-                setSelectToggle(false);
-              }}
+                type Color = 'red' | 'pink' | 'rebeccapurple' | 'grey';
```
Comment (Andrewgdewar) [inline src/pages/Landing/index.tsx]: Import the "Color" type above, don't create another type.

File: src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -95,11 +95,12 @@ export default () => {
...
+                if (typeof value === 'string' && (value === 'red' || value === 'pink' || value === 'rebeccapurple' || value === 'grey')) {
```
Comment (Andrewgdewar) [inline src/pages/Landing/index.tsx]: This shouldn't be needed.

File: src/components/ColoredTd.tsx [hunk 3]
```diff
@@ -1,15 +1,18 @@
 import { Td, TdProps } from '@patternfly/react-table';
 import { createUseStyles } from 'react-jss';
+import { choosableColors, Customer, getCustomers } from 'src/api/CustomerApi';
 
-const useStyles = (color: any) => // TODO: Use the new Color type to fix this.
+type Color = string;
```
Comment (Andrewgdewar) [inline src/components/ColoredTd.tsx]: Delete this line.

File: src/components/ColoredTd.tsx [hunk 4]
```diff
@@ -1,15 +1,18 @@
 import { Td, TdProps } from '@patternfly/react-table';
 import { createUseStyles } from 'react-jss';
+import { choosableColors, Customer, getCustomers } from 'src/api/CustomerApi';
```
Comment (Andrewgdewar) [inline src/components/ColoredTd.tsx]: choosableColors is not used here, remove it. Import "Color"

Comment (kwarnerredhat) [reply src/components/ColoredTd.tsx]: Its giving me an error

Comment (Andrewgdewar) [reply src/components/ColoredTd.tsx]: What is "its"?

Comment (Andrewgdewar) [reply src/components/ColoredTd.tsx]: I presume you mean the type you are attempting to import.
Try something like this:
`import type { SomeThing } from "./some-module.js";`

File: src/components/ColoredTd.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import { Td, TdProps } from '@patternfly/react-table';
 import { createUseStyles } from 'react-jss';
-import { choosableColors, Customer, getCustomers } from 'src/api/CustomerApi';
+import { Color as ApiColor } from 'src/api/CustomerApi';
```
Comment (Andrewgdewar) [inline src/components/ColoredTd.tsx]: So the source of your error is you were attempting to import something with the same name as a variable you created below "Color". 

You need to delete the Color type you created below (as it won't be used anyway), and import Color from above. 
Not Color "as" somethingElse

File: README.md [hunk 6]
```diff
@@ -26,7 +26,7 @@
 
 ## Tasks
 
-1. Fix the <i>color</i> variable within the Customer interface found [here](/src/api/CustomerApi.ts#L7) by creating a "Color" type, that only allows the 4 choosable colors. You will have to follow the type around to fix associated components found: [here](src/pages/Landing/index.tsx#L98) and [here](src/components/ColoredTd.tsx#L4)
+1. Fix the <i>color</i> variable within the Customer interface found [here](/src/api/CustomerApi.ts#L7) by creating a "Color" type, that only allows the 4 choosable colors. You will have to follow the type around to fix associated components found: [here](src/pages/Landing/index.tsx#L98) and [here](src/components/ColoredTd.tsx#L4).
```
Comment (Andrewgdewar) [inline README.md]: Remove the period, no real reason to add this, I was using it as an example to ensure a change (and allow a PR).

File: src/components/ColoredTd.tsx (hunk 3 above)
Comment (Andrewgdewar) [reply src/components/ColoredTd.tsx]: Delete line 5 "type Color = string;"

File: src/components/ColoredTd.tsx [hunk 7]
```diff
@@ -1,15 +1,18 @@
 import { Td, TdProps } from '@patternfly/react-table';
 import { createUseStyles } from 'react-jss';
```
Comment (Andrewgdewar) [inline src/components/ColoredTd.tsx:4]: import { Color } from 'src/api/CustomerApi';

**Review by Andrewgdewar — APPROVED** [review]
Ack

**Pull request merged**